### DISCUSSION TITLE
PostgreSQL Support

### DISCUSSION THREAD

**Discussion by Vad1mo** [ag2ai/faststream#796] [Ideas]
I am aware that PostgreSQL is now a Message Queue per se; however, it can be utilized as such. Many developers realize that it suffices for the majority of tasks, and it comes with zero added costs if a PostgreSQL db is present. 

It would be nice if [faststream](https://github.com/airtai/faststream) supported that too.

- https://adriano.fyi/posts/2023-09-24-choose-postgres-queue-technology/
- https://www.prequel.co/blog/sql-maxis-why-we-ditched-rabbitmq-and-replaced-it-with-a-postgres-queue
- https://dagster.io/blog/skip-kafka-use-postgres-message-queue

btw. 
- You can do exactly this with MySQL and SQL server too because they both support SKIP LOCKED.

**Comment by davorrunje** [comment]
Thank you for the suggestion :)

Could you please open an issue for this? We will support additional protocols by the popular demand.

**Comment by davorrunje** [comment]
Issue opened: https://github.com/airtai/faststream/issues/799
